### PR TITLE
Adds support for RHEL as a conductor container

### PR DESCRIPTION
**Pull request by jcpowermac**
##### ISSUE TYPE

 - Bugfix Pull Request

##### SUMMARY
This pull requests modifies the template `conductor-dockerfile.j2`
including a block that installs the necessary packages for a RHEL
container.

### CODE REVIEW

**Comment by chouseknecht** [comment]
@jcpowermac thanks for your work on this! I'll test it out this evening.

**Comment by jcpowermac** [comment]
@chouseknecht yw.  I tested using these updates https://github.com/RHsyseng/container-rhel-examples/pull/34 to our starter-apb image.

File: container/docker/templates/conductor-dockerfile.j2 [hunk 1]
```diff
@@ -10,6 +10,17 @@ RUN yum update -y && \
...
+{% elif "rhel" in distro %}
+RUN yum -y update-minimal --disablerepo "*" \
+                          --enablerepo rhel-7-server-rpms,rhel-7-server-optional-rpms \
+                          --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs && \
+    yum -y install --disablerepo "*" \
+                   --enablerepo rhel-7-server-rpms \
+		   https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
+    yum -y install --disablerepo "*" \
+                   --enablerepo epel,rhel-7-server-rpms,rhel-7-server-optional-rpms \
+	           gcc git python-devel rsync libffi-devel openssl-devel && \
```
Comment (chouseknecht) [inline container/docker/templates/conductor-dockerfile.j2]: I keep running into the following error during the conductor build:

```
Rpmdb checksum is invalid: dCDPT(pkg checksums): perl-HTTP-Tiny.noarch 0:0.033-3.el7 - u
```
I couldn't find a way to get yum to ignore the checksum error. The only hack I came up with is to append `; exit 0` to the end of the this whole chained command.

Comment (chouseknecht) [inline container/docker/templates/conductor-dockerfile.j2]: I keep running into the following error during the conductor build:

```
Rpmdb checksum is invalid: dCDPT(pkg checksums): perl-HTTP-Tiny.noarch 0:0.033-3.el7 - u
```
I couldn't find a way to get yum to ignore the checksum error. The only hack I came up with is to append `; exit 0` to the end of the this whole chained command.

File: container/docker/templates/conductor-dockerfile.j2 [hunk 2]
```diff
@@ -10,6 +10,17 @@ RUN yum update -y && \
...
+    yum clean all
```
Comment (chouseknecht) [inline container/docker/templates/conductor-dockerfile.j2]: I keep running into the following error during the conductor build:

```
Rpmdb checksum is invalid: dCDPT(pkg checksums): perl-HTTP-Tiny.noarch 0:0.033-3.el7 - u
```
I couldn't find a way to get yum to ignore the checksum error. The only hack I came up with is to append `; exit 0` to the end of the this whole chained command.

**Comment by jcpowermac** [comment]
@chouseknecht hmm that is weird.  My logs are attached here:
[out.txt](https://github.com/ansible/ansible-container/files/994895/out.txt)

Images
```
(testing)[root@docker ansible]# docker images
REPOSITORY                        TAG                 IMAGE ID            CREATED             SIZE
ansible-starterapb                20170512021021      1e0dfd6975fd        10 minutes ago      806.1 MB
ansible-starterapb                latest              1e0dfd6975fd        10 minutes ago      806.1 MB
ansible-conductor                 latest              731b2d147866        13 minutes ago      578.6 MB
registry.access.redhat.com/rhel   latest              a2d9f633eaab        3 weeks ago         192.7 MB
(testing)[root@docker ansible]# 
```
RHEL (updated to latest tonight)
```
(testing)[root@docker ansible]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.3 (Maipo)
```
Docker version
```
(testing)[root@docker ansible]# docker --version
Docker version 1.12.6, build 3a094bd/1.12.6
```

**Comment by chouseknecht** [comment]
I dunno. Maybe it's an issue with AWS AMI.

**Pull request merged**